com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -2796,6 +2812,7 @@ public void CopyTo(HDAdditionalLightData data)
 #endif
 
             data.UpdateAllLightValues();
+            UpdateRenderEntity();
```

Comment:
Shouldn't this be `data.UpdateRenderEntity()`?

Reply:
good catch.